projects/laji/src/app/shared-modules/datatable/service/datatable-util.service.ts

```diff
@@ -99,6 +100,21 @@ export class DatatableUtil {
           (row.scientificName || '') + (row.scientificName && row.vernacularName ? ' – ' : '') + (row.vernacularName || '')
         );
         break;
+      case template.startsWith('latestRedListEvaluation.threatenedAtArea_') ? template : !template:
+        const area = template.split('_')[1];
+        observable = ObservableOf(value.includes(area) ? 'RT' : '');
+        break;
+      case template.startsWith('latestRedListEvaluation.occurrences_') ? template : !template:
```

Comment:
never seen this kind of usage for a case :) I think you could move these two before the switch statement and then just make them basic if statements return the observables there